### ISSUE TITLE
Scrambled accents in downloadable Rmd source 

### ISSUE THREAD

**Issue by aoles** [rstudio/rmarkdown#722]
Consider the following minimal example Rmd file.

```

---
title: "Testfile"
author: "Andrzej Oleś"
output:
  html_document:
    code_download: yes

---

The document author name contains an accent.
```

When I render the document and download its source through the "Code>Download Rmd" button I get the following output:

```

---
title: "Testfile"
author: "Andrzej OleÅ"
output:
  html_document:
    code_download: yes

---

The document author name contains an accent.
```

**Comment by aoles** [comment]
Fixed in https://github.com/rstudio/rmarkdown/pull/723

**Comment by aoles** [comment]
Hi @jjallaire, @jcheng5,

I wondering whether you had a chance to look into the solutions discussed in https://github.com/rstudio/rmarkdown/pull/723. It's almost a year now since I've reported this problem, and I still observe it with the newest version of rmarkdown.

Thanks,
Andrzej

**Comment by yihui** [comment]
I have proposed a fix at #1179. You may test it with:

```r
devtools::install_github('rstudio/rmarkdown#1179')
```

Your example works okay with this fix.

**Comment by aoles** [comment]
@yihui, many thanks for taking lead on this. The fix works great for me!
